### PR TITLE
[SE-0001 ... SE-0023] Update the status, etc.

### PR DESCRIPTION
**Pull request by benrimmington**
* Status fields are updated.
* Review links are added.
* Broken links are fixed.

### CODE REVIEW

File: proposals/0001-keywords-as-argument-labels.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Allow (most) keywords as argument labels
 
-* Proposal: [SE-0001](https://github.com/apple/swift-evolution/blob/master/proposals/0001-keywords-as-argument-labels.md)
+* Proposal: [SE-0001](0001-keywords-as-argument-labels.md)
```
Comment (jessesquires) [inline proposals/0001-keywords-as-argument-labels.md]: Each of these links should probably remain complete, valid URLs. 😄

Comment (benrimmington) [inline proposals/0001-keywords-as-argument-labels.md]: @jessesquires Most of the recent proposals have those links relative to the current branch.

Comment (benrimmington) [inline proposals/0001-keywords-as-argument-labels.md]: @jessesquires The template was changed in apple/swift-evolution@2657917762864e4d17d592922bb92447048119e7

Comment (jessesquires) [inline proposals/0001-keywords-as-argument-labels.md]: 😱 👍

**Comment by benrimmington** [comment]
Closing (will conflict with #499).

**Pull request closed without merging**